docker-contributor/Dockerfile

```diff
@@ -12,7 +12,8 @@ ENV DEBIAN_FRONTEND=noninteractive \
   FPM_MAX_CHILDREN=40 \
   DJ_SKIP_MAKE=0 \
   DJ_DB_INSTALL_BARE=0 \
-  PHPSUPPORTED="7.2 7.3 8.0 8.1" \
+  PHPSUPPORTED="7.2 7.3 7.4 8.0 8.1" \
```

Comment:
I assume you know I excluded 74 as thats already in the image?

Reply:
Yes see below, i exclude it differently but I needed *all* versions in the loop and this seemed easiest.

Reply:
Yes, this is nicer.